CLA.md

```diff
@@ -37,6 +37,8 @@ Select the most appropriate option:
 
 Once you select the type of contributor, proceed to Sign the CLA and follow the instructions to complete the signing process through DocuSign.
 
+If you are signing up as a corporate contributor, ensure that you have linked your corporate email address to your GitHub profile (it doesn't have to be your primary email address for GitHub) or else it can lead to issues with the CLA system. For more information, see [Adding an email address to your GitHub account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/adding-an-email-address-to-your-github-account).
```

Comment:
Thanks! Sorry for the delay, wording wise looks good. I do have one suggestion:
We should bump this up as step 1. Folks should update their profile before going through the CLA process, we can remove the below bolded line and make it applicable to everyone.

Reply:
Thank you, PTAL.